src/utils.ts

```diff
@@ -300,3 +300,16 @@ export const compactNumber = new Intl.NumberFormat('en', {
   //@ts-ignore
   compactDisplay: 'short',
 }).format;
+
+export function isUniqueArray(arr: string[]) {
+  let map: any = {},
+    i,
+    size;
+  for (i = 0, size = arr.length; i < size; i++) {
+    if (map[arr[i]]) {
+      return false;
+    }
+    map[arr[i]] = true;
+  }
+  return true;
+}
```

Comment:
You can use `includes` method. The method already exists.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
Hecho, resuelvo la conversación...